chef/cookbooks/neutron/recipes/server.rb

```diff
@@ -149,6 +149,9 @@
 num_vlans = node[:neutron][:num_vlans]
 vlan_end = [vlan_start + num_vlans - 1, 4094].min
 
+vni_start = [node[:neutron][:vxlan][:vni_start], 0].max
+vni_end = [node[:neutron][:vxlan][:vni_end], 16777215].min
```

Comment:
Same here - if the attribute values are out of a valid range, wouldn't it be better to raise an exception, since that should already have been caught by Crowbar validation?